cmd/zed/validate.go

```diff
@@ -0,0 +1,207 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/jzelinskie/cobrautil"
+	"github.com/spf13/cobra"
+
+	v0 "github.com/authzed/authzed-go/proto/authzed/api/v0"
+	"github.com/authzed/spicedb/pkg/development"
+	"github.com/authzed/spicedb/pkg/tuple"
+	"github.com/authzed/spicedb/pkg/validationfile"
+	"github.com/charmbracelet/lipgloss"
+
+	"github.com/authzed/zed/internal/decode"
+)
+
+var (
+	success                = lipgloss.NewStyle().Bold(true).Foreground(lipgloss.Color("#00ff00")).Render("success!")
+	errorPrefix            = lipgloss.NewStyle().Bold(true).Foreground(lipgloss.Color("#ff0000")).Render("error: ")
+	errorMessageStyle      = lipgloss.NewStyle().Bold(true).Width(80)
+	linePrefixStyle        = lipgloss.NewStyle().Foreground(lipgloss.Color("#0000ff"))
+	highlightedSourceStyle = lipgloss.NewStyle().Foreground(lipgloss.Color("#ff0000"))
+	highlightedLineStyle   = lipgloss.NewStyle().Foreground(lipgloss.Color("#ff0000"))
+	codeStyle              = lipgloss.NewStyle().Foreground(lipgloss.Color("#555555"))
+	highlightedCodeStyle   = lipgloss.NewStyle().Foreground(lipgloss.Color("#ffffff"))
```

Comment:
If you're only using basic colors, use the ANSI colors: https://github.com/charmbracelet/lipgloss#ansi-256-colors-8-bit

By hardcoding it to be hex, it just makes it really hard to read for people with different color-schemes.

Reply:
ANSI color codes seem even more opaque though

Reply:
ANSI color codes use the color theme your terminal is configured to use. Hex forces absolute colors that can be entirely illegible depending on what you've configured.

Reply:
Done